src/index.js

```diff
@@ -51,7 +50,7 @@ export default class CSRF {
 
     ctx.response.__defineGetter__('csrf', () => ctx.csrf);
 
-    if (this.opts.excludedMethods.includes(ctx.method))
+    if (this.opts.excludedMethods.indexOf(ctx.method) !== 1)
```

Comment:
Can you please fix this so that it is `-1` instead?

Therefore, if the position is not `-1`, then that means it must be found.

Arrays start with key 0, not 1, but it should be -1 anyways.